foundry.toml

```diff
@@ -2,6 +2,10 @@
 src = 'src'
 out = 'out'
 libs = ['lib']
+ffi = true
+
+[fuzz]
+runs = 5
 
 # Need permission to read files
 fs_permissions = [
```

Comment:
We could probably get rid of the permissions here when we are not using them directly in that manner anymore.